components/checkbox/index.css

```diff
@@ -153,7 +153,8 @@
 		}
 	}
 
-	&:active {
+	&:active,
+	.is-active {
```

Comment:
Was this `.is-active` added for Storybook? The active state may work just fine in Storybook without it, when the dist/index.css is built I think PostCSS processes the state and adds the `.is-active` class 🙂 

Although it's fair to say that we have plenty of instances of `.is-focused` in CSS files and I'm not sure whether we need those for a different reason.